src/main/resources/application.yml

```
@@ -21,3 +21,8 @@ logging:
     org.hibernate.orm.jdbc.extract: TRACE
     org.springframework.transaction.interceptor: TRACE
     com.back: DEBUG
+
+jwt:
+  secret: "01234567890123456789012345678901"  # 최소 32자(256비트) 이상
+  access-token-validity: 1800000             # 30분 (밀리초)
+  refresh-token-validity: 604800000          # 7일 (밀리초)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)